admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/js/admin/blc-admin.js

```diff
@@ -1380,12 +1380,12 @@ var getCurrentHashVal = function() {
     return hash.substr(1);
 };
 
-$(document).ready(function() {
-    // primary entity buttons should be disabled until page is loaded
-    $(window).on('load', function () {
-        $('.button.primary.large:not(.submit-button):not(.modify-production-inventory)').prop('disabled', false).removeClass('disabled');
-    });
+// primary entity buttons should be disabled until page is loaded
+$(window).on('load', function () {
```

Comment:
is this the only place ?
I think there might couple of more places where we use onload handler, can you please doublecheck them also ?

Reply:
other places fixed in [OMS](https://github.com/BroadleafCommerce/OMS/pull/383)